src/views/playground/PlaygroundView.tsx

```diff
@@ -21,9 +22,9 @@ const PlaygroundView: React.FC<Props> = (_): JSX.Element => {
 
   const tickers = useObservableState(miniTickers$, [])
 
-  useEffect(() => {
-    console.log('tickers', tickers[0]?.s)
-  }, [tickers])
+  // useEffect(() => {
+  //   console.log('tickers', tickers[0]?.s)
+  // }, [tickers])
```

Comment:
Asking for clarification purposes - why is this included but commented out?

Reply:
Good catch - it can be removed.